content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -1251,6 +1251,7 @@ const deleteUser = await prisma.user.delete({
 #### Remarks
 
 - `createMany` is not supported by SQLite.
+- The `skipDuplicates` option is not supported by MongoDB.
```

Comment:
Is it supported on all other databases? Line 1243 only defines the other direction, and does not list explicitly. We should make this explicit and list the opposite here.